src/Styleguide/Components/Pagination.tsx

```diff
@@ -100,9 +100,16 @@ export const SmallPagination = (props: PaginationProps) => {
   )
 }
 
-const Page = ({ num, ...props }) => {
+const scrollToTop = clickHandler => {
```

Comment:
Nice, yea pagination bar only at the bottom feels odd when content refreshes and you stay there.